src/worldstate/models/syndicate_mission.rs

```diff
@@ -14,6 +14,8 @@ model_builder! {
     :"The Reward Pool of the Bounty"
     pub reward_pool: Vec<String>,
 
+    :"The name of the syndicate job"
+    pub job_name: String = "type",
```

Comment:
stop renaming stuff, dude this makes it really hard for me to know what you're looking for when you tell me there's an issue with a rust field....

Reply:
type is a reserved keyword, in addition 'type' must be the single worst name you can give to a field like this

Reply:
it's literally called `jobType` in DE's source data. i'm not gonna keep the class prefix when i translate it, bud. 

https://github.com/WFCD/warframe-worldstate-parser/blob/master/lib/models/SyndicateJob.js#L116C32-L116C44

Reply:
```suggestion
    pub job_type: String = "type",
```

since my name is dumb, use DE's dumb name